Description: fix errors reported by pod2man
 There are a couple missing =back's and =over's
Author: Damyan Ivanov <dmn@debian.org>

Index: libgd-gd2-noxpm-perl/GD.pm
===================================================================
--- libgd-gd2-noxpm-perl.orig/GD.pm	2011-07-30 09:44:27.416238216 +0200
+++ libgd-gd2-noxpm-perl/GD.pm	2011-07-30 09:45:13.033890388 +0200
@@ -460,6 +460,8 @@
 NOTE: The libgd library is unable to read certain XPM files, returning
 an all-black image instead.
 
+=back
+
 =head1 GD::Image Methods
 
 Once a GD::Image object is created, you can draw with it, copy it, and
@@ -472,6 +474,8 @@
 The following methods convert the internal drawing format into
 standard output file formats.
 
+=over
+
 =item B<$pngdata = $image-E<gt>png([$compression_level])>
 
 This returns the image data in PNG format.  You can then print it,
